recipe/vc_repack.py

```diff
@@ -247,11 +247,18 @@ def decode_manifest(directory):
 def fix_filename_and_copy(source, dest):
     cwd = os.getcwd()
     os.chdir(source)
-    for fname in glob.glob("*.dll"):
+    # as of VS 17.6, the artefact contains intermediate DLL extensions,
+    # which get renamed correctly upon installation; do it manually
+    candidates = glob.glob("*.dll") + glob.glob("*.dll_amd64")
```

Comment:
Can you do the same for `dll_arm64` for ARM?

Reply:
Of course. I thought about it but thought it's better to leave it for when it actually comes up (i.e. it is to be expected that they're called `.dll_arm64`, but as long as I haven't seen it in CI, I don't have a lot of confidence in Msft's consistency). That said, I don't mind, will do.